14 AsyncCalls_Redux_Saga/src/sagas/membersSagaWatcher.ts

```diff
@@ -0,0 +1,23 @@
+//import {takeLatest} from 'redux-saga';
+import { fork, all, takeLatest, takeEvery } from 'redux-saga/effects';
+import { fetchMembers } from './membersSaga';
+import {actionTypes} from '../common/constants/actionTypes';
+
+// add here all your watchers
+// https://github.com/yelouafi/redux-saga/blob/master/examples/real-world/sagas/index.js
+function* watchLoadMembersRequest() {
+        console.log("Estoy en members Saga llamando");
+       
+        yield takeEvery(actionTypes.FETCH_MEMBER_REQUEST, fetchMembers);
+         /* while (true){
+           let task = yield takeLatest (actionTypes.FETCH_MEMBER_REQUEST, fetchMembers);
+           console.log(task);
+        } */
```

Comment:
This needs a while true, if not it will only work the first time you launch the request.

Reply:
Nice catch. Added!